sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/SymbolicExecution/Common/SymbolicValues/NullableSymbolicValue.cs

```diff
@@ -68,31 +66,18 @@ public NullableSymbolicValue(SymbolicValue wrappedValue)
                 return new[] { programState };
             }
 
-            return TrySetConstraint(NullableValueConstraint.HasValue, programState)
-                .SelectMany(ps => WrappedValue.TrySetConstraint(constraint, ps));
+            return TrySetConstraint(NullableValueConstraint.HasValue, programState).SelectMany(ps => WrappedValue.TrySetConstraint(constraint, ps));
         }
 
         public override IEnumerable<ProgramState> TrySetOppositeConstraint(SymbolicValueConstraint constraint, ProgramState programState)
         {
             var negateConstraint = constraint?.OppositeForLogicalNot;
 
-            if (constraint is BoolConstraint)
-            {
-                return TrySetConstraint(negateConstraint, programState)
-                  .Union(TrySetConstraint(NullableValueConstraint.NoValue, programState));
-            }
-
-            return TrySetConstraint(negateConstraint, programState);
+            return constraint is BoolConstraint
+                ? TrySetConstraint(negateConstraint, programState).Union(TrySetConstraint(NullableValueConstraint.NoValue, programState))
+                : TrySetConstraint(negateConstraint, programState);
         }
 
-        public override string ToString()
-        {
-            if (base.identifier != null)
-            {
-                return $"NULLABLE_SV_{base.identifier}";
-            }
-
-            return WrappedValue?.ToString() ?? base.ToString();
-        }
+        public override string ToString() => $"NULLABLE_SV_{identifier}" ;
```

Comment:
The previous version displayed the `WrappedValue` when the `identifier` was null. This is no longer needed?

Reply:
There's no way to make `identifier` null. The `identifier` is set in base class constructor (it's the counter). This class cannot break it unless new constructor with `: base(null)` is added. So it was useless anyway and could be removed.